notebook/templates/notebook.html

```diff
@@ -84,6 +84,7 @@
                             <a href="#">Open...</a></li>
                         <!-- <hr/> -->
                         <li class="divider"></li>
+                        <li id="custom_download"><a href="#">Custom Download</a></li>
```

Comment:
I actually think moving it here is an improvement in the design in the menubar. It should not require drilling down into the menu structure to access this feature.

I'm fine moving it down next to "Download as…" but it shouldn't be one option among the various downloads as it is strictly speaking a broader class of functionality. 

Re: "Download as…" I'm thinking of actually switching them to use this post method as then they won't open a new window (which I've always disliked).